pyproject.toml

```
@@ -0,0 +1,19 @@
+[tool.mypy]
+python_version = "3.12"
+pretty = true
+strict = true
+
+# Extra checks that aren't included in --strict
+enable_error_code = "ignore-without-code"
```

These checks can also often be useful (but only enable them if they actually pass, which I haven't checked!)

```suggestion
enable_error_code = "ignore-without-code,redundant-expr,possibly-undefined,truthy-iterable"
```

---

Either fix the complaint or remove it from the list, I leave it to you ;)